doc/source/examples/metrology/Calibration With Three Receivers.ipynb

```diff
@@ -70,7 +70,20 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This example demonstrates how to create a [TwoPortOnePath](http://scikit-rf.readthedocs.org/en/latest/reference/calibration/generated/skrf.calibration.calibration.TwoPortOnePath.html#skrf.calibration.calibration.TwoPortOnePath)  and [EnhancedResponse](http://scikit-rf.readthedocs.org/en/latest/reference/calibration/generated/skrf.calibration.calibration.EnhancedResponse.html#skrf.calibration.calibration.EnhancedResponse) calibration from  measurements taken on a Agilent PNAX with a set of VDI WR-12 TXRX-RX Frequency Extender heads.  Comparisons between the two algorithms are made by correcting an asymmetric  DUT."
+    "This example demonstrates how to create a [TwoPortOnePath](../../api/calibration/generated/skrf.calibration.calibration.TwoPortOnePath.rst) and [EnhancedResponse](../../api/calibration/generated/skrf.calibration.calibration.EnhancedResponse.rst) calibration from  measurements taken on a Agilent PNAX with a set of VDI WR-12 TXRX-RX Frequency Extender heads.  Comparisons between the two algorithms are made by correcting an asymmetric  DUT."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Measure the Calibration Standands and DUT\n",
+    "\n",
+    "First, one measures the S-parameters of the Open, Short, and Load standards on the VNA according to the standard SOLT calibration procedures. We assume the reader is familiar with this task in scikit-rf. If not, it's fully described in the [SOLT example](./SOLT.ipynb).\n",
+    "\n",
+    "When calibrating a three-receiver VNA, the process is almost identical. But note that there are a few minor differences. First, the Open, Short, and Load standards are only measured at port 1, not port 2. The VNA cannot measure in the reverse orientation, so there's no reason to perform OSL calibration at port 2 (unless one needs the optional isolation calibration). Nevetheless, scikit-rf still expects two-port networks as input, we therefore still save all results as two-port networks - only $S_{11}$ and $S_{21}$ are the actual measurements - $S_{12}$ and $S_{22}$ contain no meaningful data, but it's okay.\n",
```

Comment:
To clarify even more: S12 and S22 will be overridden in the `TwoPortOnePath` constructor to make the incomplete 2-port compatible for the `TwelveTerm` routine (by making it symmetrical with S22=S11, S12=S21). https://github.com/scikit-rf/scikit-rf/blob/7ad561755afbb6c939baa4c68a0d49b96d4c5684/skrf/calibration/calibration.py#L1825-L1829

So, the user can put any data they want for S12 and S22, most people will probably chose S12=S22=0, or they can directly put S12=S21, S22=S11. It does not matter.

Reply:
I just changed the wording to:

> we therefore still save all results as two-port networks. In this case, only $S_{11}$ and $S_{21}$ are the actual measurements, $S_{12}$ and $S_{22}$ are unused, not meaningful, and can contain arbitrary data. One can use all-zero values as placeholders. Internally, scikit-rf will set $S_{22} = S_{11}$ and $S_{12} = S_{21}$ for the purpose of calculations.